arm_base:
  object_mesh: "package://task_ui/meshes/wine_bottle_reference.dae"
  num_task_poses: 2
  reference_frame: "world"

  num_robot_placements: 10

  object_start_pose:  [1.5, 0.5, 0.8, 1.5708, 0.0, 0.0]
  object_goal_pose:  [1.5, 0.4, 0.8, 1.5708, 0.0, 0.0]  

